protocol-designer/src/molecules/InputStepFormField/index.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  COLORS,
+  DIRECTION_COLUMN,
+  Flex,
+  Icon,
+  InputField,
+  SPACING,
+  StyledText,
+  Tooltip,
+  useHoverTooltip,
+} from '@opentrons/components'
+import { getFieldDefaultTooltip } from '../../components/StepEditForm/utils'
+
+import type { FieldProps } from '../../components/StepEditForm/types'
+
+interface InputStepFormFieldProps extends FieldProps {
+  title: string
+  units: string
+}
+
+export function InputStepFormField(
+  props: InputStepFormFieldProps
+): JSX.Element {
+  const {
+    errorToShow,
+    onFieldBlur,
+    onFieldFocus,
+    updateValue,
+    value,
+    name,
+    title,
+    ...otherProps
+  } = props
+  const { t } = useTranslation(['tooltip', 'application'])
+  const [targetProps, tooltipProps] = useHoverTooltip()
+
+  return (
+    <Flex flexDirection={DIRECTION_COLUMN} padding={SPACING.spacing16}>
+      {title !== null ? (
```

Comment:
maybe 👇 because of `title: string`

```suggestion
      {title !== '' ? (
```